src/Build/Collections/RetrievableEntryHashSet/HashHelpers.SerializationInfoTable.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
from Hashset, untested here. And hopefully going away soon with your use of BinaryFormatter.